testsuite/integration-tests/pom.xml

```diff
@@ -68,6 +68,11 @@
             <artifactId>smallrye-config</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jboss.resteasy</groupId>
+            <artifactId>resteasy-client-vertx</artifactId>
+            <version>${version.org.jboss.resteasy}</version>
+        </dependency>
```

Comment:
This should be defined in the resteasy-microprofile-test-bom and have a `<scope>test</scope>`

Reply:
@jamezp Okay! I'll work on it.